CLONE_RESTORE.md

```diff
@@ -0,0 +1,200 @@
+<h1>Creating and Restoring in-cluster Postgres Database Backups in Harboor</h1>
+
+<h2>Why do we want to  create backups</h2>
+- To be able to recover harbor indexed data in case of disaster so we can still have the list of cached artifacts stored in harbor
+
+There are currently two ways to configure a database in harbor
+1. Standard database configurations can be used to set the existing pre-deployed or cloud database services as the dependent database of the deploying Harbor.
+2. The in-cluster database configuration can be configured to let the <b>Harbor operator automatically deploy an in-cluster PostgreSQL database service using the zalando postgres operator</b> with HA supported as the dependent database of the deploying Harbor. 
+
+The second method is the preferred method because removes the need to manage a separate postgres instance.
+
+<h2> Creating and storing backups in S3 using WALG </h2> 
+In other to create backups stored in s3 the following fields must be set in the `pod-config` configmap the postgres operator will read environment variables from.
+
+```
+USE_WALG_BACKUP: "true"
+USE_WALG_RESTORE: "true"
+BACKUP_NUM_TO_RETAIN: "14" # For 2 backups per day, keep 7 days of base backups
+BACKUP_SCHEDULE: "*/10 * * * *"
+AWS_ACCESS_KEY_ID: < * >
+AWS_SECRET_ACCESS_KEY: < * >
```

Comment:
perhaps we should be setting a [secret](https://github.com/giantswarm/harbor-operator-app/blob/a1333bd7599ddfa8f5c7fb6ce26dc777b84fa7b3/helm/harbor-operator/charts/postgres-operator/values.yaml#L126) rather than a configmap for these values